docs/ruleset_functions/get_primary_secondary_loops.md

```diff
@@ -0,0 +1,71 @@
+
+## get_primary_secondary_loops
+
+Description: Get the list of primary and secondary loops for HHW or CHW for a RMR.
+
+Inputs:  
+- **RMR**: The RMR that needs to get the list of primary and secondary loops.
+- **loop_type**: "COOLING" (or "HEATING", if needed in future)
+
```

Comment:
As it stands now, the function only applies to cooling (e.g., it looks at chillers but not boilers) so I think we either need to remove references to heating here (and perhaps update name of the function), or expand the function to also cover heating loops.